src/go/rpk/pkg/cli/cmd/redpanda.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package cmd
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/redpanda"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	rp "github.com/vectorizedio/redpanda/src/go/rpk/pkg/redpanda"
+)
+
+func NewRedpandaCommand(
+	fs afero.Fs, mgr config.Manager, launcher rp.Launcher,
+) *cobra.Command {
+	command := &cobra.Command{
+		Use:	"redpanda",
+		Short:	"Interact with a local node",
```

Comment:
Node is a bit of a loaded term. Can we say "Interact with a local Redpanda process"?

Reply:
Sure.